src/components/composite/common/cards/user-card/UserCard.tsx

```diff
@@ -87,15 +92,15 @@ const UserCard: FC<UserCardProps> = ({ avatarSrc, displayName, city, country, ta
           <CardContent className={styles.cardContent}>
             <Grid container spacing={1}>
               <Grid item>
-                <Typography color="textPrimary" variant={'h3'} noWrap={true}>
+                <Typography color="primary" variant={'h5'} noWrap={true} fontWeight={600}>
```

Comment:
should we avoid using our Typography component?

Reply:
No. We should stop using the `core/Typography` component in favor of the material's one.